.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,23 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  storybook:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Where does this version get defined, do you know? Is it using the [one defined here](https://github.com/kumar303/skeleton-kit/blob/main/.github/workflows/test.yaml#L16)?

Reply:
Doubt it has anything to do with test.yml.  There is some magic GH does in the CI stuff that I'm not a huge fan of.  I imagine it's just the default version of node they use on their CI image, which is probably latest stable version.  So it's continually being updated which is what you want I think.

Reply:
By "their CI image" I mean the default one when you specify `uses: actions/setup-node@v1`.